### ISSUE TITLE
Help understanding how to read the code

### ISSUE THREAD

**Issue by ryanmaxwell96** [pat-coady/trpo#30]
Hello,

Just a quick question. In policy.py in class Policy it uses the Keras package to call "get_layer". This is the output layer correct? Also, I sent an email out so feel free to ignore this part if you already answered it, but I see from the TRPO paper that the NN is supposed to only calculate the mean and somehow uses another set of parameters which are a vector of the same size as the number of actions. But the paper is not clear to me how stdev is actually computed or updated. And in this code, all of it is computed under the hood in Keras.

Anyhelp on this would be greatly appreciated!

Ryan

**Comment by ryanmaxwell96** [comment]
Also, can you help me understand why there are multiple neural network outputs? I tested the halfcheetah code and found out that the observation is a (1,27) but for some reason a (1,6) vector of means is returned and I'm at a loss as to why there are 6 means being returned.

**Comment by ryanmaxwell96** [comment]
Unless it refers to the 6 half-cheetah joints that can move depending on the state (of 27 dimensions). So depending on which of these states it is in, the policy will tell it what position each of these joints should be in via means and log vars.

**Comment by pat-coady** [comment]
Exactly, the policy network takes a state and returns an action. The dimensionality of state and action are different. I'd have to look for sure, but my guess is the cheetah only accepts 6 actuation inputs. But many more positions and velocities are measured on the cheetah.

**Comment by ryanmaxwell96** [comment]
Ok thank you. Sorry I have another question. Where is the name "policy_nn" coming from? I'm guessing it is the last layer, correct?

**Comment by ryanmaxwell96** [comment]
I've been trying to modify the code and in doing so have been trying to construct the NN layers for both policy and value function in a different file. But it doesn't like this and I'm not sure how to do this correctly

**Comment by ryanmaxwell96** [comment]
Can you please explain to me why in value.py it has an output Dense layer of size 1? Shouldn't it be the same size as the action dimension?

**Comment by ryanmaxwell96** [comment]
Also, how do you use plotting.py? I don't see it currently being used in any of the code.
